app/models/mail_handler.rb

```diff
@@ -126,6 +126,16 @@ def options=(value)
     options[:no_permission_check] = options[:no_permission_check].to_s == '1'
   end
 
+  def reply_to_wp_from_receiver_addresses
+    addresses = [email.to, email.cc].flatten.compact.uniq.map { |a| a.strip.downcase }
+    unless addresses.empty?
+      addressMatch = addresses.find { |addr| /\+(\d+)/ =~ addr }
+      unless addressMatch.nil?
+        receive_work_package_reply(addressMatch.match(/\d+/).to_s)
```

Comment:
The addresses are parsed two times here which makes the code harder to understand. The better approach would be to parse the addresses for the matching `/\+(\d+)/` segment (ideally only on the emission email address) and have that returned by a method.